src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -251,38 +238,46 @@ function build_adios2
             cfg_opts="${cfg_opts} -DCMAKE_PREFIX_PATH:PATH=${hdf5_install_path}"
         fi
 
-        # call configure.
+        #
+        # Several platforms have had problems with the VTK cmake configure command
+        # issued simply via "issue_command". This was first discovered on
+        # BGQ and then showed up in random cases for both OSX and Linux machines.
+        # Brad resolved this on BGQ  with a simple work around - we write a simple
+        # script that we invoke with bash which calls cmake with all of the properly
+        # arguments. We are now using this strategy for all platforms.
+        #
```

Comment:
Would it make sense to embody this as a `helper_func` and just have *all* modules do it this way? It would be more *consistent*.

Reply:
Sure it would, but again all bv_ file would need to be edited.